js/styles/atlas.css

```diff
@@ -347,17 +340,17 @@ div#wrapperMainWindowContainer {}
 
 .feFilter {
 	padding-left: 0px;
+	padding-top: 99px;
 }
 
 .fa.fa-refresh.fa-spin {
 	font-weight: normal;
 }
 
```

Comment:
made changes to the appearance of the facets along side components to align the facets with the table it was filtering